.pre-commit-config.yaml

```
@@ -0,0 +1,24 @@
+repos:
+  - repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: v1.3.0
```

```suggestion
    rev: v2.1.0
```

The current version on the graphene repo is v2.1.0